book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
In the Book class, kindly make sure that there's a method for adding rentals where the `book` itself is sent as a parameter to create the new rental achieving the 'has-many' association.

For example:

```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

Remember to pay attention to the order of arguments in your Rental class. The code above is just a suggestion, therefore, you must modify it to fit the flow of your application.